frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsage.tsx

```diff
@@ -175,123 +48,17 @@ const TrafficInfoBoxes = styled('div')(({ theme }) => ({
     gap: theme.spacing(2, 4),
 }));
 
-const BoldText = styled('span')(({ theme }) => ({
-    fontWeight: 'bold',
-}));
-
-const useTrafficStats = (
-    includedTraffic: number,
-    chartDataSelection: ChartDataSelection,
-) => {
-    const consumptionModelEnabled = useUiFlag('consumptionModel');
```

Comment:
we used to have traffic stats and consumption stats mixed together. I created 2 separate hooks used in two different tabs